src/map.rs

```diff
@@ -16,11 +18,19 @@ use core::{
 /// map.entry("hello".to_string()).and_modify(|mut v| v.push_str("!"));
 /// assert_eq!(map.get("hello").map(String::as_str), Some("world!"))
 /// ```
-#[derive(Clone, Default, PartialEq, Eq)]
+#[derive(Clone, PartialEq, Eq)]
 pub struct Map<K, V> {
     backing: Vec<(K, V)>,
 }
 
+impl<K, V> Default for Map<K, V> {
+    fn default() -> Self {
+        Self {
+            backing: Vec::default(),
+        }
+    }
+}
+
```

Comment:
This allows for `Map::default()` event if `K` or `V` do not implement `Default`.